### PR TITLE
Add Azure and Google Cloud repo URL in docs reference

### PR DESCRIPTION
**Pull request by rajkan01**
- Added Azure and Google Cloud repo URL under the cloud connector docs section.

### CODE REVIEW

File: docs/introduction/tutorials_intro.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ These tutorials show you how to install, export a project to and start a debuggi
 
 - [Pelion Device Management](https://github.com/ARMmbed/mbed-os-example-pelion)
 - [AWS MQTT broker](https://github.com/ARMmbed/mbed-os-example-for-aws)
+- [Azure IoT Hub](https://github.com/ARMmbed/mbed-os-example-for-azure)
+- [Google Cloud IoT core](https://github.com/ARMmbed/mbed-os-example-for-google-iot-cloud)
```
Comment (evedon) [inline docs/introduction/tutorials_intro.md]:
```suggestion
- [Mbed OS example for Google IoT Cloud](https://github.com/ARMmbed/mbed-os-example-for-google-iot-cloud)
```

File: docs/introduction/tutorials_intro.md [hunk 2]
```diff
@@ -56,6 +56,8 @@ These tutorials show you how to install, export a project to and start a debuggi
 
 - [Pelion Device Management](https://github.com/ARMmbed/mbed-os-example-pelion)
 - [AWS MQTT broker](https://github.com/ARMmbed/mbed-os-example-for-aws)
+- [Azure IoT Hub](https://github.com/ARMmbed/mbed-os-example-for-azure)
```
Comment (evedon) [inline docs/introduction/tutorials_intro.md]:
```suggestion
- [Mbed OS example for Azure IoT Hub](https://github.com/ARMmbed/mbed-os-example-for-azure)
```

File: docs/introduction/tutorials_intro.md [hunk 3]
```diff
@@ -56,6 +56,8 @@ These tutorials show you how to install, export a project to and start a debuggi
 
 - [Pelion Device Management](https://github.com/ARMmbed/mbed-os-example-pelion)
 - [AWS MQTT broker](https://github.com/ARMmbed/mbed-os-example-for-aws)
```
Comment (evedon) [inline docs/introduction/tutorials_intro.md]:
```suggestion
- [Mbed OS example for AWS IoT SDK](https://github.com/ARMmbed/mbed-os-example-for-aws)
```

**Review by evedon — APPROVED** [review]

**Review by Elise-Kaminski — APPROVED** [review]
Looks good to me. Thanks.

**Pull request merged**